### DISCUSSION TITLE
Support 3.10.8

### DISCUSSION THREAD

**Discussion by chadknutson** [rabbitmq/rabbitmq-message-timestamp#36] [General]
I'm getting error when enabling for 3.10.8

`$ rabbitmq-plugins enable rabbitmq_routing_node_stamp

Enabling plugins on node rabbit@my-server-01:
rabbitmq_routing_node_stamp
Error:
Failed to enable some plugins: 
    rabbitmq_routing_node_stamp:
        Plugin doesn't support current server version. Actual broker version: "3.10.8", supported by the plugin: ["3.10.x-3.10.x"]`

**Comment by michaelklishin** [comment]
[This 3.10.x release](https://github.com/rabbitmq/rabbitmq-message-timestamp/releases/tag/v3.10.0) works perfectly fine for me against 3.10.9 on Erlang 25:

```
./sbin/rabbitmq-plugins enable rabbitmq_message_timestamp
Enabling plugins on node rabbit@sunnyside:
rabbitmq_message_timestamp
The following plugins have been configured:
  rabbitmq_message_timestamp
Applying plugin configuration to rabbit@sunnyside...
The following plugins have been enabled:
  rabbitmq_message_timestamp

started 1 plugins.
```
